### DISCUSSION TITLE
Fix the last command

### DISCUSSION THREAD

**Discussion by riccardodivirgilio** [TheR1D/shell_gpt#261] [Ideas]
It would be great if you could simply ask help for the last failed command.

Let's say I wrote some rsync command with some of the flag wrong, the command would fail with an exit code.

by running something like "sgpt --fix" the language model could suggest a variant of your command that would work.

Thanks.

**Comment by endolith** [comment]
Better if you could respond with a comment. "I don't think that's right, because x"
